samples/dashboards/demo/minimal/demo.css

```diff
@@ -6,6 +6,67 @@
     display: none;
 }
 
-#title h1 {
+h1#title {
+    padding-top: 10px;
+    margin: 0;
+    background-color: #3d3d3d;
     text-align: center;
+    color: white;
+}
+
+#kpi-vitamin-a .highcharts-dashboards-component-title::before,
+#kpi-iron .highcharts-dashboards-component-title::before {
+    width: 14px;
+    height: 14px;
+    border-radius: 28px;
+    display: inline-block;
+    padding: 0;
+    margin-right: 4px;
+    background-color: var(--highcharts-color-0);
+    content: " ";
+}
+
+#kpi-iron .highcharts-dashboards-component-title::before {
+    background-color: var(--highcharts-color-1);
+}
+
+#kpi-vitamin-a .highcharts-dashboards-component-kpi-subtitle,
+#kpi-iron .highcharts-dashboards-component-kpi-subtitle {
+    margin-top: 10px;
+    font-size: 1.2em;
+    color: var(--highcharts-neutral-color-60);
+}
+
+#kpi-vitamin-a .highcharts-dashboards-component-kpi-value,
+#kpi-iron .highcharts-dashboards-component-kpi-value {
+    font-size: 4em;
+    font-weight: normal;
+    line-height: 2.4rem;
+    margin-top: 10px;
+    color: var(--highcharts-color-0);
+}
+
+#kpi-iron .highcharts-dashboards-component-kpi-value {
+    color: var(--highcharts-color-1);
```

Comment:
The iron value color is very low contrast against the background in dark mode. We should probably not try to use the same text colors between light/dark, as that is hard to make work.